dll/directx/wine/ddraw/ddraw.c

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
This line should not be changed. That can be achieved by using an editor, that doesn't touch the ByteOrderMark, e.g. Notepad++.
You must not add BOM.